config/Sage/jobs.txt

```diff
@@ -30,17 +30,17 @@ Chaplain=1,1
 Station Engineer=6,5
 Atmospheric Technician=4,2
 
-Medical Doctor=7,4
+Medical Doctor=8,4
 Chemist=3,2
 Geneticist=3,2
 Virologist=1,1
 
-Scientist=6,4
+Scientist=8,4
 Roboticist=2,2
 
 Warden=1,1
 Detective=1,1
-Security Officer=5,5
+Security Officer=7,5
```

Comment:
Delta doesn't have the equipment for this.

Reply:
im gonna make it 6 then

Reply:
Check for the locker landmarks though to see if I'm mistaken.